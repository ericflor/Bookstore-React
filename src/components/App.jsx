import './App.css';
import axios from 'axios';
import Layout from './layout/Layout';
import BookContainer from './book/BookContainer';

const App = () => {

  axios(`http://localhost:8080/books`).then(response => {console.log(response.data)});

  return (
    <Layout>
      <BookContainer></BookContainer>
    </Layout>
  );
}

export default App;
