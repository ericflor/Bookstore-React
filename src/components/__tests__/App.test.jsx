import App from '../App';
import React from 'react';
import renderWithRedux from '../../util/testUtil';

describe("App component", () => {

  it("should render without crashing", () => {

    const {asFragment} = renderWithRedux(<App />, {});

    expect(asFragment()).toMatchSnapshot();

  });
})
