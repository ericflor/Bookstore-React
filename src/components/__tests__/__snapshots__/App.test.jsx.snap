// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component should render without crashing 1`] = `
<DocumentFragment>
  <div
    class="MuiBox-root css-0"
  >
    <div
      class="MuiBox-root css-0"
    >
      <header
        class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionFixed mui-fixed css-11b3ww9-MuiPaper-root-MuiAppBar-root"
      >
        <div
          class="MuiToolbar-root MuiToolbar-gutters MuiToolbar-dense css-10ltm04-MuiToolbar-root"
        >
          <h6
            class="MuiTypography-root MuiTypography-h6 css-1qp5wg9-MuiTypography-root"
          >
            Bookstore
          </h6>
        </div>
      </header>
    </div>
    <div
      class="MuiBox-root css-1afphko"
    >
      <div
        class="book-container MuiBox-root css-0"
      >
        <div
          class="book-filter MuiBox-root css-0"
        >
          <div
            class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 book-filter-paper css-1ps6pg7-MuiPaper-root"
          >
            book filter
          </div>
        </div>
        <div
          class="book-list MuiBox-root css-0"
        >
          <div
            class="book-list MuiBox-root css-0"
          />
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
